public/app/plugins/datasource/cloudwatch/dataquery.cue

```diff
@@ -121,9 +121,8 @@ composableKinds: DataQuery: {
 						#QueryEditorPropertyType: "string" @cuetsy(kind="enum")
 
 						#QueryEditorArrayExpression: {
-							type: (#QueryEditorExpressionType & "and") | (#QueryEditorExpressionType & "or")
-							// TS type expressions: QueryEditorExpression[] | QueryEditorArrayExpression[], extended in veneer
-							expressions: _
+							type:        (#QueryEditorExpressionType & "and") | (#QueryEditorExpressionType & "or")
+							expressions: [...#QueryEditorExpression] | [...#QueryEditorArrayExpression]
 						} @cuetsy(kind="interface")
 
 						// QueryEditorArrayExpression is added in veneer
```

Comment:
now that `QueryEditorArrayExpression` is removed from `public/app/plugins/datasource/cloudwatch/expressions.ts`, can this comment be removed?

Reply:
This comment is still valid as we need to add `QueryEditorArrayExpression` for the QueryEditorExpression [here](https://github.com/grafana/grafana/pull/66664/files#diff-14207fbfed980793148579cfe9216fa38939e690de05b16c53e3b4c55d4db080R29). This is related to the linked bug with nested recursive types.